front-end/README.md

```diff
@@ -61,3 +63,11 @@
 * **Fastclick**: removes 300ms delay on touch devices [source](https://github.com/ftlabs/fastclick)
 * **Modernizr**: feature testing [source](https://modernizr.com/)
 * **HammerJS**: adds touch gestures [source](http://hammerjs.github.io/)
+* **VelocityJS**: accelerated JavaScript animations with or without jQuery [source](http://velocityjs.org/)
+* **Waypoints**: useful for events triggered by scroll points [source](http://imakewebthings.com/waypoints/)
+* **Underscore**: great set of utility functions (including debounce and throttle) [source](http://underscorejs.org/)
```

Comment:
i would include lodash or swap it with underscore. it is the larger library overall, but it is modular and lets us import only the functions we need, which results in less overhead once compiled in webpack/browserify. Also I think it is the superior library to underscore and is better optimized.